src/Universum/Monad/Reexport.hs

```diff
@@ -69,14 +69,16 @@ import Control.Monad hiding (fail)
 #if __GLASGOW_HASKELL__ >= 800
 import Control.Monad.Fail (MonadFail (..))
 #else
-import Prelude (Maybe (Nothing), String)
+import Prelude (String)
 import Text.ParserCombinators.ReadP (ReadP)
 import Text.ParserCombinators.ReadPrec (ReadPrec)
 
 import qualified Prelude as P (fail)
 #endif
 
 #if __GLASGOW_HASKELL__ < 800
```

Comment:
This `#if` is going exactly after previous `#endif`. I think we can merge two `if`s into single `if-else-endif`.